### DISCUSSION TITLE
Add var like `meow-keypad-describe-delay` for `meow-display-thing-help`

### DISCUSSION THREAD

**Discussion by eki3z** [meow-edit/meow#364] [Ideas]
Could you provide same delay-display feature for display-thing too or apply the var for all descriptions at the same time

**Comment by DogLooksGood** [comment]
They are implemented in a different way. But you can use `meow-display-thing-help` variable to control whether to display the help table.

**Reply by eki3z** [reply]
@DogLooksGood thanks. I'm trying to do some hack.
